src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java

```diff
@@ -43,6 +43,81 @@
   private static final Pattern LONG_DATE =
       Pattern.compile("^(\\d{4})-(\\d{2})-(\\d{2})T(\\d{2}):(\\d{2}):(\\d{2})Z$");
 
+  private static final Map<String, String> CONTENT_FORMATTER = getContentFormatter();
+
+  private static Map<String, String> getContentFormatter() {
+    Map<String, String> contentFormatter = new LinkedHashMap<>();
```

Comment:
not a big deal... but curious how you chose `LinkedHashMap` instead of just `HashMap`. I would suggest using HashMap unless you have a particular reason since it raises eyebrows. If you do have a reason, suggest commenting what that reason is.

Reply:
Raising eyebrows is def not my goal. 😂  Thanks for calling this out!

Reply:
Do we know how we are going to consume this map? If we are reading key by key (all keys), LinkedHashMap will make sense.

Reply:
nitpick: I think only reason to use LinkedHashMap is if you want to preserve the order of the map. Anyways both will work just fine for what you're doing. :shipit: